### ISSUE TITLE
Better Docs for "unavailable" Error

### ISSUE THREAD

**Issue by miqrogroove** [friendica/friendica#5241]
### Expected behavior

Need helpful instructions any time Friendica generates a fatal error.

### Actual behavior

"System is currently unavailable. Please try again later"

Looking for help in Google, it only brings me to INSTALL.txt, which says "Check your database settings."  This is not helpful.

Through brute experimentation, I found it was necessary to execute a SQL command to restore my website:

```UPDATE `config` SET `v` = 100 WHERE `k` = 'maxloadavg_frontend`;```

I tried other values as high as 90 and nothing else worked for me.  This needs to be explained as a possible solution, perhaps here on the wiki.

### Friendica version you encountered the problem

2018.08-dev

**Comment by MrPetovan** [comment]
I agree the message isn't helpful, but we have to balance it with the fact that we probably shouldn't give out too much information publicly when the site goes out.

When your site is out because of the load, you should get a meaningful line in your log, provided you enabled it in advance, of course.

But the same screen shows up when the database can't be reached, and I'm not sure we have a meaningful message in the log for that.

**Comment by miqrogroove** [comment]
I wish I knew what to do with "Active worker limit reached".  According to the ps command in my user shell, there is nothing running.  The locks table looks mostly empty.  Where is it finding these workers?

**Comment by annando** [comment]
That message about the worker limit is okay. The "Load:" line above tells you that your system currently has got a load of 31.43 and you set a maximum load of 100. Question: Are you on some shared host or some non Linux system? A load of > 30 is really, really high - and for Linux systems this would mean that your system is overloaded.

Because of the high load only a single worker out of 3 possible workers will be spawned. This is the value "1/3" from above. The just spawned worker is quitting because the limit has been reached.

I suggest raising the number of active workers to 10. This can be done in the admin interface. Additionally please check which load values are okay for your system. The maximum load (that is currently set to "100") should be set to a value where your system is at a very high load where it would be best to not do anything more to calm down the system.

**Comment by miqrogroove** [comment]
I'm getting load numbers around 70 or even higher.

As you said, "The just spawned worker is quitting because the limit has been reached."  In other words, when the limit is reduced to 1, nothing runs ever.  Isn't that a bug in the worker design?

**Comment by AlfredSK** [comment]
> I'm getting load numbers around 110 or even higher.

What? :-) That's impossible. Unless you have at least a 50+ cores CPU. On a single core a load of 1.0 means 100% CPU utilization.

**Comment by miqrogroove** [comment]
Not impossible at all.  But yes, it's a shared host and I can only imagine what goes on behind the scenes there.

**Comment by annando** [comment]
Possibly you should set the maximum number of workers to a value of "200" and increase the maximum number of workers to 20. Then I guess that enough workers will be spawned.

**Comment by miqrogroove** [comment]
Good suggestions there.  What I've done so far is set the worker max load to 90, escalated the insane load factor problem to the hostmaster, and wrote up Issue #5243 separately with a new patch.
